Server/Controllers/MatchController.cs

```diff
@@ -197,13 +203,14 @@ List<string> GetSmartRandomHeroes(bool onMap)
 
                         return player;
                     })
-                    .ToList()
+                    .ToList(),
+                LeaderBoard = topPlayers
```

Comment:
```suggestion
                Leaderboard = customGame == CustomGame.Dota12v12 ? await _ratingService.GetLeaderboard() : null,
```

Reply:
again, this is a bad practice to do everything in one line. Hard to do debugging and not that readable. I've changed it nevertheless